back-end/src/base/calls/calls.controller.ts

```diff
@@ -30,7 +30,6 @@ export class CallsController {
 		return res.status(201).json({ status: 'ok', qrcode: qrcode });
 	}
 	@Get('/presence/:jwt')
-	@UseGuards(JwtAuthGuard)
```

Comment:
The reason ?

Reply:
You're using your phone, as you're not logged in you cannot interact with this route if you keep the @UseGuards.